views/render-identify.php

```diff
@@ -0,0 +1,18 @@
+<script type="text/javascript">
+    (function() {
+        const f = function () {
+            metrilo.identify("<?php echo $this->identify_call_data; ?>");
+            <?php
+            if ($this->user_tags) {
+                echo "window.metrilo.applyTags($this->user_tags);";
```

Comment:
I see that `$this->user_tags` is defined as an array:
```php
private $user_tags = [];
```
https://github.com/Metrilo/metrilo-woocommerce-integration/pull/49/files#diff-eb7ba8216233b4ab2c51f481a3636445af65ce5e77690e211ab1388ce2c1e191R14

Then we assign it something from wordpress/woocommerce:
```php
$this->user_tags = $user_roles;
```
https://github.com/Metrilo/metrilo-woocommerce-integration/pull/49/files#diff-eb7ba8216233b4ab2c51f481a3636445af65ce5e77690e211ab1388ce2c1e191R174

If the assignment is also with an array, the evaluation to string will fail.

Reply:
Good point. Will json format the array before printing the event.